app/uk/gov/hmrc/customs/declaration/services/fileTransmissionCallbackToXmlNotification.scala

```diff
@@ -0,0 +1,50 @@
+/*
+ * Copyright 2018 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package uk.gov.hmrc.customs.declaration.services
+
+import javax.inject.Singleton
+
+import uk.gov.hmrc.customs.declaration.model.{FileTransmissionFailureNotification, FileTransmissionSuccessNotification}
+
+import scala.xml.NodeSeq
+
```

Comment:
See previous comment. I think this can be one function in one class

Reply:
Done